modules/machine-pool/variables.tf

```diff
@@ -78,3 +78,9 @@ variable "upgrade_acknowledgements_for" {
   description = "Indicates acknowledgement of agreements required to upgrade the cluster version between minor versions (e.g. a value of \"4.12\" indicates acknowledgement of any agreements required to upgrade to OpenShift 4.12.z from 4.11 or before)."
 }
 
+variable "tuning_configs" {
+  type = list(string)
+  default = null
+  description = "Allows to supply pre created tuning configs for a machine pool"
```

Comment:
@gdbranco I think it's worth being clearer that this is the list of the`name` of tuningconfig, so perhaps something like:

```
description = "A list of tuning config names to attach to this machine pool. The tuning configs must already exist".
```